db/fixtures/miq_event_definitions_sets.yml

```diff
@@ -0,0 +1,800 @@
+---
```

Comment:
I think you typo'd the file name - think this should be `miq_event_definitions.yml`